CoffeeChat/Controller/NoMatchViewController.swift

```diff
@@ -30,7 +30,7 @@ class NoMatchViewController: UIViewController {
         let imageBottomPadding: CGFloat = LayoutHelper.shared.getCustomVerticalPadding(size: 36)
         let imageWidth = (UIScreen.main.bounds.width / 375) * 176
         let profileButtonSize = CGSize(width: 35, height: 35)
-        let subtitleLabelPadding: CGFloat = LayoutHelper.shared.getCustomVerticalPadding(size: 24)
+        let subtitleLabelPadding: CGFloat = Constants.Onboarding.skipBottomPadding
```

Comment:
can't leave a comment below -- but is there a specific reason why the constraints are all done in `viewDidLoad` instead of in a constraints function? if not, seeing that most of the other files that I've seen have a `setupConstraints` function, can we do that in this file too?

Reply:
Oh lol its because we had kind of gone with the convention of if the setup is short enough, we won't make a separate function for it.. not sure which convention we want to stick with